explorer/src/lib/components/data-card/DataCard.svelte

```diff
@@ -37,7 +39,9 @@
       <Button
         on:click={headerButtonDetails.action}
         text={headerButtonDetails.label}
-        variant={headerButtonDetails.variant || "secondary"}
+        variant={darkMode
+          ? "tertiary"
+          : headerButtonDetails.variant || "secondary"}
```

Comment:
Check my comment on the button variants and theming.

The main idea is we should not change the variant semantically based on the theme. Instead, we should tweak the appearance of the variant to match the dark theme, as variants like "primary", "secondary", "danger", etc., are defined based on their roles and importance in the UI, not their visual appearance.